react/features/base/ui/components/native/Button.tsx

```diff
@@ -28,13 +29,17 @@ const Button: React.FC<IProps> = ({
     labelKey,
     labelStyle,
     onClick: onPress,
+    rippleColor = true,
```

Comment:
This prop name suggests a color, not a boolean. How about `useRippleColor` ?

Reply:
I agree, will change.